README.md

```diff
@@ -75,10 +94,23 @@ When all texts are translated, generate the locale artifacts:
 npm run locale:build
 ```
 
-## Responsive Design
+## Authentication via OAuth 2.0
 
-Mobile breakpoints:
+The _Evento Portal_ is handling the authentication using the the [OAuth 2.0 Authorization Code Flow mit Proof Key for Code Exchange (PKCE)](https://clx-evento.bitbucket.io/master_eventodoc/Api/Autorisierung/Login-OAuth-Server/#authorization-code-flow-mit-proof-key-for-code-exchange-pkce). It makes sure the user is authenticated and an access token is available in sessionStorage as under the `CLX.LoginToken` key to be used by the _apps_.
 
-- > 1920px content centered
-- < 1200px hamburger menu
-- < 767px mobile compact
+The flow works as follows:
+
+- A random _code verifier_ is generated and remembered in sessionStorage. Then the user is redirected to the [OAuth login page](https://clx-evento.bitbucket.io/master_eventodoc/Api/Autorisierung/Login-OAuth-Server/#login-mit-dem-formular) with a _code challenge_ (i.e. a hash of the _code verifier_).
+- The user is redirected back from the login page to the _Evento Portal_ with a resulting _code_.
+- The _Evento Portal_ exchanges this _code_ together with the original _code verifier_ for an access & refresh token using the [token endpoint](https://clx-evento.bitbucket.io/master_eventodoc/Api/Autorisierung/Login-OAuth-Server/#authorization-code-flow-mit-proof-key-for-code-exchange-pkce) (XHR). The tokens are then stored in localStorage.
+- Before access tokens expire, the _Evento Portal_ renews them using the [refresh token endpoint](https://clx-evento.bitbucket.io/master_eventodoc/Api/Autorisierung/RefreshToken/).
+
+### About Tokens & Scopes
+
+The access tokens are bound to a specific [OAuth scope](https://clx-evento.bitbucket.io/master_eventodoc/Api/Autorisierung/ApplicationScopes_Intro/) is specified when getting a token via login page or refresh token endpoint. Each _app_ requires a certain scope since the scope determines what the user can access.
```

Comment:
"[...] are bound to [...] is [...]" -> in diesem Satz ist etwas komisch :)

Reply:
Ups, da fehlt noch ein «that».